### PR TITLE
Fix Duplicated Solutions and Improve Library Load Time

### PR DESCRIPTION
**Pull request by wenchuanchen**
TODO:
1. How to check duplicated solutions?
2. Can we add this check to CI?
3. If CI can do this, then remove workaround

### CODE REVIEW

File: library/src/amd_detail/rocblaslt/src/tensile_host.cpp [hunk 1]
```diff
@@ -2190,13 +2190,6 @@ rocblaslt_status getAllSolutions(MyProblem&
     int i = 0;
     for(auto solution : solutions)
     {
-        //workaround: findAllSolutions should get all solutions without duplications
```
Comment (jichangjichang) [inline library/src/amd_detail/rocblaslt/src/tensile_host.cpp]: Consider if any way we can add test to check there is no duplicated solution.

**Comment by wenchuanchen** [comment]
TODO:
1. how to check duplicated solutions?
2. can we add this check to CI?
3. If CI can do this, then remove workaround

**Review by jichangjichang — APPROVED** [review]
conditional approve, please keep following the TODO items

**Review by hcman2 — APPROVED** [review]

**Review by Jinp800125 — APPROVED** [review]

**Comment by wenchuanchen** [comment]
For gfx90x
./hipblaslt-test --gtest_filter=*pre_checkin*

[----------] Global test environment tear-down
[==========] 13005 tests from 12 test suites ran. (1798292 ms total)
[  PASSED  ] 13005 tests.

**Pull request merged**